app/src/app/map/page.tsx

```diff
@@ -16,6 +17,7 @@ export default function Map() {
           <MapComponent />
           <MobileTopNav />
           <MapContextMenu />
+          <ErrorNotification />
```

Comment:
Q: Should this be further up in the DOM in case other pages eventually need it?

Reply:
Possibly but I'm not sure if that will mess up SSR stuff